src/server.c

```diff
@@ -321,6 +324,11 @@ int main(int argc, char **argv) {
         break;
       case 'a':
         server->url_arg = true;
+        server->arg_file = NULL;
```

Comment:
i think we might want to add a comment in help text, documentation that if both are set, behaviour is non deterministic.

Alternatively we can make a choice that if both are set url_arg will win (or vice versa)

Based on that you want to structure your `if .. else if ..` in https://github.com/tsl0922/ttyd/pull/747/files#diff-0b6794e089b51873f4effd373d78e8d13d1ee1ca83c0dd1394b09bcad26254c7R105
(eitherways a comment will be helpful)

Reply:
The behaviour shouldn't be non deterministic here; the last specified argument of `-f` and `-a` will be set. I can add this to the help text for clarity.